storage/replica.go

```diff
@@ -193,11 +196,12 @@ var _ client.Sender = &Replica{}
 // NewReplica initializes the replica using the given metadata.
 func NewReplica(desc *roachpb.RangeDescriptor, rm *Store) (*Replica, error) {
 	r := &Replica{
-		store:       rm,
-		cmdQ:        NewCommandQueue(),
-		tsCache:     NewTimestampCache(rm.Clock()),
-		sequence:    NewSequenceCache(desc.RangeID),
-		pendingCmds: map[cmdIDKey]*pendingCmd{},
+		store:          rm,
+		cmdQ:           NewCommandQueue(),
+		tsCache:        NewTimestampCache(rm.Clock()),
+		sequence:       NewSequenceCache(desc.RangeID),
+		pendingCmds:    map[cmdIDKey]*pendingCmd{},
+		raftEntryCache: newRaftEntryCache(defaultRaftEntryCacheSize),
```

Comment:
Even 100 per Replica seems high. I'm worried that when a replica is down (preventing log truncation), memory usage will jump much higher than usual, and since this is per-replica we don't have a global limit on how much memory it could use.